recipes/osx-sysroot/conda_build_config.yaml

```diff
@@ -0,0 +1,51 @@
+_MACOSX_DEPLOYMENT_TARGET_:
+  - "10.9"
+  - "10.13"
+  - "11.0"
+  - "11.0"
+  - "12.3"
+
+cross_target_platform:
+  - osx-64
+  - osx-64
+  - osx-64
+  - osx-arm64
+  - osx-arm64
```

Comment:
Same here; I'd like to keep this in sync between platforms (post 11.0). I think we could do something like
```
_MACOSX_DEPLOYMENT_TARGET_:
  - "10.9"     # [osx and x86_64]
  - "10.13"    # [osx and x86_64]
  - "11.0"
  - "12.3"     
```
and then not zip it with cross_target_platform. Also, we currently have MACOSX_DEPLOYMENT_TARGET's in use that don't fall into this scheme (I guess they can take the next one up?).

Reply:
This recipe is noarch so platform dependent selectors won't work. We could build it on each platform I suppose.

Reply:
I plan to add the rest of the versions once we all agree on the recipe format.